internal/pkg/cli/local_run.go

```diff
@@ -0,0 +1,336 @@
+		deployStore:        deployStore,
+	}
+	opts.envVersionGetter = func(envName string) (versionGetter, error) {
+		envDescriber, err := describe.NewEnvDescriber(describe.NewEnvDescriberConfig{
+			App:         opts.appName,
+			Env:         envName,
+			ConfigStore: opts.store,
+		})
+		if err != nil {
+			return nil, fmt.Errorf("new environment compatibility checker: %v", err)
+		}
+		return envDescriber, nil
+	}
+	return opts, nil
+}
+
+func (o *localRunOpts) Validate() error {
+	if o.appName == "" {
+		return errNoAppInWorkspace
+	}
+	if _, err := o.store.GetApplication(o.appName); err != nil {
+		return fmt.Errorf("get application %s: %w", o.appName, err)
+	}
+	return nil
+}
+
+func (o *localRunOpts) Ask() error {
+	if err := o.validateOrAskWorkloadName(); err != nil {
+		return err
+	}
+	if err := o.validateOrAskEnvName(); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (o *localRunOpts) Execute() error {
+	taskDef, err := o.ecsLocalClient.TaskDefinition(o.appName, o.envName, o.wkldName)
+	if err != nil {
+		return fmt.Errorf("get task definition: %w", err)
+	}
+
+	awsSession := session.Must(session.NewSessionWithOptions(
+		session.Options{
+			SharedConfigState: session.SharedConfigEnable,
+		}))
+
+	_, err = o.getDecryptedSSMSecrets(taskDef, awsSession)
+	if err != nil {
+		fmt.Println("unable to get secrets from SSM store")
+	}
+
+	_, err = o.getDecryptedSecretManagerSecrets(awsSession)
+	if err != nil {
+		fmt.Println("unable to get secrets from Secret Manager")
+	}
+
+	return nil
+}
+
+func (o *localRunOpts) getDecryptedSSMSecrets(taskDef *awsecs.TaskDefinition, awsSession *session.Session) ([]ssmSecret, error) {
+	var SSMSecretsList []ssmSecret
+	secrets := taskDef.Secrets()
+	ssmClient := ssm.New(awsSession)
+	for _, secret := range secrets {
+		resp, err :=
+			ssmClient.GetParameter(&ssm.GetParameterInput{
+				Name:           aws.String(secret.ValueFrom),
+				WithDecryption: aws.Bool(true),
+			})
+		if err != nil {
+			return nil, err
+		}
+		secretInfo := ssmSecret{
+			Name:  secret.Name,
+			Value: *resp.Parameter.Value,
+		}
+		SSMSecretsList = append(SSMSecretsList, secretInfo)
+	}
+	return SSMSecretsList, nil
+}
+
+func (o *localRunOpts) getDecryptedSecretManagerSecrets(awsSession *session.Session) ([]secretManager, error) {
+	secretsManagerClient := secretsmanager.New(awsSession)
+	var secretStruct secretManager
+	var SecretManagerSecrets []secretManager
+	input := &secretsmanager.ListSecretsInput{}
+	err := secretsManagerClient.ListSecretsPages(input, func(page *secretsmanager.ListSecretsOutput, lastPage bool) bool {
+		for _, secret := range page.SecretList {
+			secretName := *secret.Name
+			in := &secretsmanager.GetSecretValueInput{
+				SecretId:     aws.String(secretName),
+				VersionStage: aws.String("AWSCURRENT"),
+			}
+			result, _ := secretsManagerClient.GetSecretValue(in)
+
+			secretValue := aws.StringValue(result.SecretString)
+
+			_ = json.Unmarshal([]byte(secretValue), &secretStruct)
```

Comment:
why do we swallow the error here? If we _know for sure_ that there won't be an unmarshal error, can we have a comment here that says so 🥺

Reply:
In addition, I think we should initialize a new `secretStruct` for individual secrets instead of reusing the same one from all secrets. 

We don't want this to happen:
```go
// say secret1's JSON string looks like  `{"name": "firstsecret", "value": "1"}`
// and secret2's JSON string looks like   `{"name": "secondsecret"}`
var secretStruct secretManager
_ = json.Unmarshal([]byte(secret1), &secretStruct)
// secretStruct is {Name: "firstsecret", Value: "1"}

_ = json.Unmarshal([]byte(secret2), &secretStruct)
// secretStruct is {Name: "secondsecret", Value: "1"} but it should be {Name: "secondsecret", Value: ""}
```